great-idea/css/index.css

```diff
@@ -63,4 +63,396 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+nav{
+	display: flex;
+	padding: 10px;
+	width: 100%;
+	justify-content: space-around;
+	align-items:center;
+	flex-wrap: wrap;
+}
+nav a{
+	padding:10px;
+	margin:10px;
+	text-decoration: none;
+	color: gray;
+	box-sizing: border-box;
+	text-align: : center;
+	font-size:16px;
+}
+
+nav img{
+	padding: 10px;
+	margin:10px;
+}
+
+.hero-section{
+	width:100%;
+	height: 550px;
+	display: flex;
+	justify-content:space-around;
+	align-items: center;
+	flex-wrap: wrap;
+	border-bottom: 1px solid #717171
+}
+
+.hero-section .copy{
+	width: 470px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	flex-wrap: nowrap;
+}
+
+.hero-section  h1{
+	font-size: 85px;
+	text-align: center;
+	width: 300px;
+}
+
+.hero-section img{
+	width:500px;
+	height: 500px;
+}
+
+.hero-section button{
+	width: 200px;
+	font-size:16px;
+	height: 50px;
+	background: white;
+	border-style: solid;
+	border-radius: : 2px;
+	border-color: black;
+}
+
+.intro{
+	width: 100%;
+	height: 275px;
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+}
+
+.intro h2{
+	font-size: 24px;
+}
+.intro .features{
+	width: 500px;
+	margin: 20px;
+	padding: 20px;
+}
+.intro .about{
+	width: 500px;
+	margin:20px;
+	padding: 20px;
+}
+
+.middle-img{
+	display: flex;
+	justify-content:center;
+	align-items: center;
+	width: 100%
+}
+
+
+
+.details{
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	border-bottom: 1px solid #717171;
+}
+
+.details h2{
+	font-size: 24px;
+}
+
+.details .services{
+	width: 250px;
+	margin: 20px;
+	padding: 20px;
+}
+
+
+.details .products{
+	width: 250px;
+	margin: 20px;
+	padding: 20px;
+
+}
+.details .vision{
+	width: 250px;
+	margin: 20px;
+	padding: 20px;
+}
+footer{
+	width: 100%;
+	margin-left: 80px;
+	padding: 50px;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+
+}
+
+span{
+	display: flex;
+	width: 100%;
+	justify-content: center;
+	align-items: center;
+	margin: 10px;
+	padding: 10px;
+	align-self: center;
+}
+
+
+footer p{
+	line-height: 25px;
+}
+
+footer address{
+	padding-bottom: 20px;
+}
+
+footer h2{
+	font-size: 24px;
+}
+
+.Services-nav nav{
+	display: flex;
+	padding: 10px;
+	justify-content: space-around;
+	align-items:center;
+	flex-wrap: wrap;
+}
+.Services-nav a{
+	padding:5px;
+	margin:5px;
+	text-decoration: none;
+	color: gray;
+	box-sizing: border-box;
+	text-align: : center;
```

Comment:
Be careful of syntax errors like this. Here, there should only be one colon (`:`). Is there linting support in Atom?